WordPress/build.gradle

```diff
@@ -469,7 +469,7 @@ android.buildTypes.all { buildType ->
     }
 }
 
-task violationCommentsToGitHub(type: se.bjurr.violations.comments.github.plugin.gradle.ViolationCommentsToGitHubTask) {
+tasks.register("violationCommentsToGitHub", se.bjurr.violations.comments.github.plugin.gradle.ViolationCommentsToGitHubTask) {
```

Comment:
Just typing this out to verify I understood it correctly from the docs: By using the `tasks.register("<name>"` API instead of `task <name>` we are now registering the task, as opposed of creating it. This way, if there aren't tasks in the dependency tree that depend on this one, it won't be allocated.

Neat. But, why doesn't Gradle figure out which to allocate based on the dependency tree automatically?

Reply:
That's correct. I don't know exactly why they needed a different syntax for the lazy configuration as opposed to the eager one. If I had to guess, since this is a behavior change and porting it to the new system is not always as straightforward as this one, they needed to give developers a way to differentiate the two 🤷 

Also, not all eager task configurations are supported, so some tasks need to be re-written to take advantage of the task configuration avoidance.